.ci/deps.sh

```diff
@@ -94,6 +94,15 @@ if [[ -z "$(which hlint)" ]]; then
   sudo dpkg -i $hlint_deb
 fi
 
+# Get ghc-mod if not installed
+if [[ -z "$(which ghc-mod)" ]]; then
```

Comment:
@vijeth-aradhya can we elimintate this if we upgrade circle to a newer version? if so, please leave a comment to remove it when we inplement trusty circle